docs/sections/ci_features.md

```diff
@@ -0,0 +1,16 @@
+# CI features
+
+## Specifying which workflows to run via commit messages
```

Comment:
We can move this to be a subsubsection of https://popper.readthedocs.io/en/latest/sections/cli_features.html#continuously-validating-a-pipeline